pkgs/by-name/ws/wslay/package.nix

```
@@ -12,15 +22,19 @@ stdenv.mkDerivation rec {
   };
 
   strictDeps = true;
-
-  nativeBuildInputs = [ cmake ];
-
-  cmakeFlags = [
-    (lib.cmakeBool "WSLAY_TESTS" true)
-  ];
+  postPatch = ''
+    sed -i 's!add_stylesheet!add_css_file!' doc/sphinx/conf.py.in
+  '';
+  configurePhase = ''
+    ${autoconf}/bin/autoreconf -i
+    ${automake}/bin/automake
+    ${autoconf}/bin/autoconf
+    ./configure --prefix $out
+  '';
+
+  nativeBuildInputs = [ autoconf automake libtool pkg-config sphinx nettle ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook pkg-config sphinx nettle ];
```

Also, I doubt if `nettle` is really used, since `strictDeps` is enabled here and `nettle` has no binaries.

---

It works after I put `nettle` in `buildInputs`. Thank you for your suggestion.